src/util/message.h

```diff
@@ -44,9 +49,44 @@ struct messaget
     template <typename... Args>
     static void println(FILE *f, VerbosityLevel lvl, Args &&...args)
     {
-      if(lvl == VerbosityLevel::Error)
-        fmt::print(f, "ERROR: ");
-      fmt::print(f, std::forward<Args>(args)...);
+      if(ENABLE_COLOR)
+      {
+        switch(lvl)
+        {
+        case VerbosityLevel::Error:
+          fmt::print(
+            f, fmt::fg(fmt::color::red) | fmt::emphasis::bold, "[ERROR] ");
```

Comment:
Does it have to be in brackets? Compared to a simple `:`, to me this is visual clutter. When colors are disabled, this still is a simple `:` prefix, which is why the regression tests go through.